src/obsidian_to_org/__main__.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```

Comment:
**Question:** What's the motivation for this change?

Reply:
Please refer to this: https://stackoverflow.com/questions/43793040/how-does-usr-bin-env-work-in-a-linux-shebang-line. It makes the script more portable across different environments